hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -979,6 +979,8 @@
   /*
    * cluster replication constants.
    */
+  public static final String REPLICATION_OFFLOAD_ENABLE_KEY = "hbase.replication.offload.enabled";
+  public static final boolean REPLICATION_OFFLOAD_ENABLE_DEFAULT = false;
```

Comment:
Only used in ReplicationSourceManager so define these there?

Reply:
In the next patches, this may be used in other place, too.